clang/include/clang/Sema/TypoCorrection.h

```diff
@@ -282,7 +282,7 @@ class CorrectionCandidateCallback {
 public:
   static const unsigned InvalidDistance = TypoCorrection::InvalidDistance;
 
-  explicit CorrectionCandidateCallback(IdentifierInfo *Typo = nullptr,
+  explicit CorrectionCandidateCallback(const IdentifierInfo *Typo = nullptr,
```

Comment:
Improved const correctness -- I love it!! :-) (If the review takes a while, feel free to land the const correctness changes as an NFC commit.)